### PR TITLE
Slither

### PR DESCRIPTION
**Pull request by SkandaBhat**
FYI Slither also has a WIP vscode integration - https://github.com/samparsky/slither-vscode

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -84,29 +84,17 @@ jobs:
           path: coverage
           prefix: coverage
 
-  mythx:
+  eth-security-toolbox:
```
Comment (hrishikeshio) [inline .circleci/config.yml:87]: `slither` would be more appropriate name as the toolbox contains many tools

Comment (SkandaBhat) [reply .circleci/config.yml:87]: The docker image we use for this job gives us access to other tools as well. 

https://github.com/trailofbits/eth-security-toolbox

The intention was to expand this job to include one or more tools from the toolbox in the future. If we are not planning on using them, then seems reasonable to call it slither. What do you think?

Comment (hrishikeshio) [reply .circleci/config.yml:87]: this job is only running slither for now. Better to name it slither.

Comment (SkandaBhat) [reply .circleci/config.yml:87]: Sure. 👍

Comment (SkandaBhat) [reply .circleci/config.yml]: @hrishikeshio pls resolve and merge

**Review by hrishikeshio — APPROVED** [review]

**Pull request merged**